packages/sdk/src/adapters/multipeer/rules.ts

```diff
@@ -257,35 +257,75 @@ export const Rules: { [id in Payloads.PayloadType]: Rule } = {
             beforeQueueMessageForClient: (
                 session: Session,
                 client: Client,
-                message: Message,
+                message: Message<Payloads.ActorCorrection>,
                 promise: ExportedPromise
             ) => {
-                message.payload.type = 'actor-update';
-                client.queueMessage(message, promise);
-                return undefined;
+                // Coalesce this actor correction with the previously queued update if it exists, maintaining a single
+                // update for this actor rather than queuing a series of them.  This is fine, as we do not need to lerp
+                // an actor correction on a late join user.  It can just be the updated actor values.
+                const payload = message.payload;
+                const queuedMessage = client.queuedMessages
+                    .filter(value =>
```

Comment:
>filter [](start = 21, length = 6)

Why not a simple `find` instead of `filter` and `shift`?

Reply:
I simply replicated what was already wrote for the actor-update rule.  @eanders-MS would be a better person to answer his original reasoning in doing it this way.

Reply:
An oversight on my part for not commenting it initially. Thanks for correcting it.